test/conformance/api/v1/service_validation_test.go

```diff
@@ -7,7 +7,7 @@ Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
 
-    http://www.apache.org/licenses/LICENSE-2.0
+	http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Can we leave this file unchanged and move the changes to a new PR (if you feel like they're necessary)